lab/bonus-07-use-azd-template.md

```diff
@@ -1,35 +1,35 @@
 # Bonus: Using AZD + GitHub Actions to Manage your Deployment
-If you want to use this as the starting point for your next django project, we have made this an template available directly in the Azure Developer CLI.
 
-## Deploying AZD Via Template
+You can also redeploy your project to Azure using GitHub Actions. This will allow you to redeploy your project to Azure when changes are made without manually running `AZD deploy`.
```

Comment:
```suggestion
You can also redeploy your project to Azure using GitHub Actions. This will allow GitHub to redeploy your project to Azure when changes are made without manually running `azd deploy`.
```